content/en/docs/gke/deploy/management-setup.md

```diff
@@ -37,8 +37,23 @@ For a more detailed explanation of the changes affecting Kubeflow 1.1 on Google
 
 1. Install [Kustomize v3.2.1](https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv3.2.1).
 
-    Note, Kubeflow is not compatible with later versions of Kustomize. Read [this GitHub issue](https://github.com/kubeflow/manifests/issues/538) for the latest status.
+    **Note:** Kubeflow is not compatible with Kustomize versions above 3.2.1. Read [this GitHub issue](https://github.com/kubeflow/manifests/issues/538) for the latest status.
 
+1. Next, you'll be installing `yq`. Here are some options:
+
+    * Follow the [official instructions](https://github.com/mikefarah/yq#install).
+    
+    * Use Go:
+
+       1. Check that you have [Go](https://golang.org) installed by running the `go version`
+          command. If you don't, you can download a binary from
```

Comment:
Sorry, I might not be clear enough, I suggest moving the download a binary instruction as a top level option.

Because right now without context, it sounds like the binary we download is a go binary, also it sounds like after downloading the yq binary, I still need to install yq from go.